Fortinet/fortigate/tests/antivirus.CEF.json

```diff
@@ -31,7 +31,13 @@
     "destination": {
       "address": "2.2.2.2",
       "ip": "2.2.2.2",
-      "port": 80
+      "port": 80,
+      "registered_domain": "http://2.2.2.2/eicar.com"
```

Comment:
http://2.2.2.2 is not a domain 

https://www.elastic.co/guide/en/ecs/current/ecs-dns.html#field-dns-question-registered-domain